src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureAnalyzer.cs

```diff
@@ -417,11 +417,6 @@ public override void Initialize(AnalysisContext context)
                 }
             }
 
-            if (ProcessTypeAttributeForPreviewness(symbol, requiresPreviewFeaturesSymbols, previewFeatureAttributeSymbol, out SyntaxReference? attributeSyntaxReference, out string? attributeName, out ISymbol? previewAttributeSymbol))
```

Comment:
@333fred I see that the test `Microsoft.NetCore.Analyzers.Runtime.UnitTests.DetectPreviewFeatureUnitTests.TestCustomAttribute` is last remaining test failure on the PR. Do you want to take a look?

Reply:
I'm going to skip the test and ask @buyaa-n to take a look. The failure is intermittent, presumably a race of some kind. It is not reproing when running just that test, but does when more tests are run. I'm also failing to repro it under the debugger after trying a number of times.

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/6134 is the bug.